pyproject.toml

```diff
@@ -1,39 +1,71 @@
 [tool.poetry]
 name = "tap-thinkific"
 version = "0.0.1"
-description = "`tap-thinkific` is a Singer tap for thinkific, built with the Meltano SDK for Singer Taps."
-authors = ["Ben White"]
+description = "Singer tap for Thinkific, built with the Meltano Singer SDK."
+readme = "README.md"
+authors = ["Ben White <ben.white@birdie.care>"]
 keywords = [
     "ELT",
-    "thinkific",
+    "Thinkific",
 ]
-license = "Apache 2.0"
+classifiers = [
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
```

Comment:
Why do we need to specify all of these Python versions_ If that's a requirement, maybe we need to set only >3.11 as older versions have vulnerabilities, right?